app/models/group.js

```diff
@@ -181,5 +187,24 @@ exports.addModel = function(database) {
     })
   }
 
+  /**
+   * Checks if the specified user can post to the timeline of this group.
+   */
+  Group.prototype.validateCanPost = function(postingUser) {
```

Comment:
I like that flow! There are three more ideas we can discuss: 
- authorization could be an extra layer in ./app/authorizers that will list a pre-defined set of actions: canCreate, canRead, canUpdate, and canDestroy
- controller actions are mapped to authorization verbs, i.e. PostsController#create -> Group.prototype.canCreate(user)
- there are static authorizers (a user can read any Group, a user can create any Group) and prototype authorizers (a user can destroy a group, a user can update a group).